src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public static class ScriptConstants
+    {
+        public const string AzureFunctionsWebHookContextKey = "MS_AzureFunctionsWebHookContext";
+
+        public const string AzureFunctionsHttpResponseKey = "MS_AzureFunctionsHttpResponse";
+
+        public const string DefaultSystemTriggerParameterName = "<AF>systemTrigger";
```

Comment:
Good point about the dashboard. Yes, we do add the context as a system parameter, and I agree that it we should be consistent if we were to take this approach.

I'll simplify the name as you suggested and open an issue to track this. There are some enhancements we need to make (to both, this and the context) in case the user wants to take an `ExecutionContext` or an `HttpRequestMessage` anyway, so I'll have those tracked as part of the same issue.